apps/model_eval_viz/app.py

```diff
@@ -98,21 +98,58 @@ def _get_data(_filepath:str)->gpd.GeoDataFrame:
     options=state_list,
 )
 
-# @pn.depends(state=' '.join(state_selector.param.value))
+
+base_map_options = {
+    'OpenStreetMap': gv.tile_sources.OSM,
+    'ESRI Imagery': gv.tile_sources.EsriImagery,
+    'ESRI World Street Map': gv.tile_sources.EsriWorldStreetMap,
+}
+
+map_selector = pn.widgets.Select(
+    description="Use to select Base Map",
+    name="Select a Base Map",
+    options=list(base_map_options.keys()),
+
+)
+
+
+def display_map(map:list=map_selector.value)->gv.tile_sources:
```

Comment:
The argument `map` does not need a default value and should be of type `str`. It will also return a type of gv.WMTS (checked using the `type` function)

```python
def display_map(map: str) -> gv.WMTS: